packages.yml

```diff
@@ -6,7 +6,7 @@ packages:
     version: 0.7.2
```

Comment:
I think snowplow just had a number of new updates pushed as well if you could update!